### PR TITLE
Update readme to point to preview registration experience

### PR DESCRIPTION
**Pull request by navyasric**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,24 +35,25 @@ To use this sample, you will need :
 
 ## Register the application
 
-1. Sign into the [App Registration Portal](https://apps.dev.microsoft.com/) using either your personal or work or school account.
+1. Sign in to the [Azure Portal](https://portal.azure.com/) to register an application.
 
-1. Choose **Add an app**.
+1. If your account gives you access to more than one tenant, select your account in the top right corner, and set your portal session to the desired Azure AD tenant.
```
Comment (jmprieur) [inline README.md:40]: (click on **Switch directory** and then choose the tenant)

**Review by jmprieur — APPROVED** [review]
LGTM.
Added a suggestion

File: README.md [hunk 2]
```diff
@@ -35,24 +35,25 @@ To use this sample, you will need :
...
-1. Choose **Web**.
+1. Select the **Web** platform under the **Redirect URI** section and set the value to `https://localhost:44302/`.
 
-1. Make sure the **Allow Implicit Flow** check box is selected, and enter *https://localhost:44302/* as the Redirect URI. 
+1. When finished, select **Register**.  On the app **Overview** page, note down the **Application ID** value.
 
-1. Choose **Save**.
+1. This quickstart requires the [Implicit grant flow](https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-oauth2-implicit-grant-flow) to be enabled. In the left-hand navigation pane of the registered application, select **Authentication**.
 
+1. In **Advanced settings**, under **Implicit grant**, enable both **ID tokens** and **Access tokens** checkboxes. ID tokens and Access tokens are required since this app needs to sign in users and call an API.
```
Comment (jmprieur) [inline README.md:54]: you might want to use italic (only one star) for the labels, and bold (2 stars) for the button, input text, combo box, checkboxes: controls that require an action?

**Review by jmprieur — APPROVED** [review]
LGTM
I've added a suggestion

File: README.md (hunk 2 above)
Comment (navyasric) [reply README.md:54]: Thanks for the review @jmprieur. I'm keeping it consistent with the Docs page per suggestion from Content team to bold these.

File: README.md (hunk 1 above)
Comment (navyasric) [reply README.md:40]: Thanks for the review. I'm following the recommendation from Lesia and Celeste to be consistent with the Docs article.

**Pull request merged**